### DISCUSSION TITLE
resume/

### DISCUSSION THREAD

**Discussion by giscus** [jeffabailey/comments#2] [General]
# resume/

Are you looking for a Software Development Leader with Diverse Experience? Let's build a fantastic product together!

https://jeffbailey.us/resume/

**Comment by rrrutledge** [comment]
I've worked with Jeff at Nike and the InnerSource Commons.  One thing I can say for certain is that Jeff steps up to get things done.  If you want a team that moves forward, you want Jeff on the team.
